.gitignore

```diff
@@ -1,7 +1,8 @@
-.vs
-.vscode
-out
-build
-.idea
-cmake-build*
-install
\ No newline at end of file
+.vs
+.vscode
+out
+build
+.idea
+cmake-build*
+install
+.cache

```

Comment:
I think files are mix of LF and CRLF and my vscode setup did this automatically.